### DISCUSSION TITLE
Gihub Yolo Badge not received

### DISCUSSION THREAD

**Discussion by Priyanshu-Singh-1** [community/community#38547] [Other Feature Feedback, Questions, & Ideas]
Hello community,

I am recently having an issue that I have not receive the yolo badge on my account even though I have merged the PR as per github guidelines.

I am just asking for knowledge and I am not hunting for that badge 🙃

**Comment by mdAshik922** [comment]
Pull request, Merged, comment or code review writing

**Reply by Priyanshu-Singh-1** [reply]
I have already done that 🤕 but have not received it yet...

**Reply by mdAshik922** [reply]
Keep trying from different angles. Hope your wish come true sometime. Thank you.

**Comment by SilasRodrigues19** [comment]
@Priyanshu-Singh-1 

Try creating a new branch in your repo and open a PR, after that you merge without assigning reviewers

**Reply by Priyanshu-Singh-1** [reply]
I have created a new branch and have also merged the PR without any reviews.....but still it doesnot worked...

Have they changed the criteria?!?

**Reply by SilasRodrigues19** [reply]
I don't think so, mine was exactly like that. I made changes to a minor branch and merged it into the main branch, without reviewers

**Comment by sevbanbayrak** [comment]
It may come after a certain period. How long have you been waiting?

**Reply by Priyanshu-Singh-1** [reply]
I have been waiting for the month and I have also created the new branch and have accepted the PR without any review...But still I have not received yet.

Can you help me out!!

**Comment by david-engelmann** [comment]
Was the unreviewed PR into a main branch?

**Comment by radar-blip** [comment]
Have you tried clicking on the Achievements link below your profile picture, maybe you have it and its just hidden from view.
If thats the case, just unhide it.

**Reply by Priyanshu-Singh-1** [reply]
I have even checked that also....

Yet I have not received it

**Comment by WMRamadan** [comment]
Follow these steps:
1 - Create a PR from your account.
2 - Assign a reviewer to the PR and this reviewer cannot be yourself. 
3 - Merge the PR from your account without the approval of the reviewer.
